v8go.cc

```diff
@@ -1010,6 +1031,25 @@ RtnValue ObjectGet(ValuePtr ptr, const char* key) {
   return rtn;
 }
 
+RtnValue ObjectGetInternal(ValuePtr ptr, uint32_t idx) {
```

Comment:
I can merge this with `ObjectGetIdx`, since they do the same thing, besides line 1038.